.github/workflows/ci_windows.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: CI - Windows
+
+on:
+  schedule:
+    # Weekdays at 13:00 UTC = 05:00 PST / 06:00 PDT.
+    - cron: "5 4 * * 1-5"
+  workflow_dispatch:
+
+concurrency:
+  # A PR number if a pull request and otherwise the commit hash. This cancels
+  # queued and in-progress runs for the same PR (presubmit) or commit
+  # (postsubmit). The workflow name is prepended to avoid conflicts between
+  # different workflows.
+  group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  build_test_all_windows:
```

Comment:
Sample logs from my fork: https://github.com/ScottTodd/iree/actions/runs/10034629151/job/27729434331

If we have enough of these nightly jobs taking 4h+ on GitHub-hosted runners then we may want to stagger them so they don't use up our limit of 20 concurrent actions (https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits)

(that had multiple jobs all grouped into a single "nightly" workflow)